package/autoyast2.changes

```diff
@@ -1,3 +1,11 @@
+-------------------------------------------------------------------
+Thu Feb  1 12:59:08 CET 2018 - schubi@suse.de
+
+- Reporting packages which cannot be selected for installation.
```

Comment:
What about somthing like "Report packages which cannot be select for installation (except those packages not included in the AutoYaST profile)" or something similar?